common/metrics/tags.go

```diff
@@ -283,3 +288,24 @@ func (d statsTypeTag) Key() string {
 func (d statsTypeTag) Value() string {
 	return d.value
 }
+
+// Returns a new stats type tag
+func CustomTag(key string, value string) Tag {
```

Comment:
I guess the whole point of this file is to avoid `customTag`. I think you should better define two concrete tags.